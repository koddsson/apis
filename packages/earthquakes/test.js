/* global expect, it, describe, jest */

jest.mock("request");

const getEarthquakes = require("./index.js");

describe("getting earthquakes", () => {
  it("should show all the earthquakes that are happening", () => {
    return getEarthquakes().then(earthquakes => {
      expect(earthquakes).toMatchSnapshot();
    });
  });
});
