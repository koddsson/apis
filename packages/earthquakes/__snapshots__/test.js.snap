// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getting earthquakes should show all the earthquakes that are happening 1`] = `
Array [
  Object {
    "depth": "10.8",
    "humanReadableLocation": "7.1 km NA af Flatey",
    "latitude": "66.214",
    "longitude": "-17.736",
    "quality": "77.67",
    "size": "1.0",
    "timestamp": "2017-05-18T19:05:38.000Z",
  },
  Object {
    "depth": "3.5",
    "humanReadableLocation": "16.8 km ANA  af Kistufelli",
    "latitude": "64.828",
    "longitude": "-16.871",
    "quality": "33.36",
    "size": "0.5",
    "timestamp": "2017-05-18T17:24:40.000Z",
  },
  Object {
    "depth": "10.0",
    "humanReadableLocation": "15.0 km ANA  af Kistufelli",
    "latitude": "64.812",
    "longitude": "-16.899",
    "quality": "65.75",
    "size": "0.7",
    "timestamp": "2017-05-18T17:04:50.000Z",
  },
  Object {
    "depth": "2.6",
    "humanReadableLocation": "1.6 km VSV af Kröfluvirkjun",
    "latitude": "65.706",
    "longitude": "-16.786",
    "quality": "59.54",
    "size": "0.3",
    "timestamp": "2017-05-18T17:00:55.000Z",
  },
  Object {
    "depth": "12.6",
    "humanReadableLocation": "6.7 km NA af Flatey",
    "latitude": "66.214",
    "longitude": "-17.751",
    "quality": "52.68",
    "size": "1.0",
    "timestamp": "2017-05-18T16:33:07.000Z",
  },
  Object {
    "depth": "5.6",
    "humanReadableLocation": "34.7 km ASA  af Grímsey",
    "latitude": "66.410",
    "longitude": "-17.299",
    "quality": "34.31",
    "size": "1.5",
    "timestamp": "2017-05-18T13:43:49.000Z",
  },
  Object {
    "depth": "5.6",
    "humanReadableLocation": "4.4 km V af Dreka",
    "latitude": "65.037",
    "longitude": "-16.699",
    "quality": "99.0",
    "size": "0.6",
    "timestamp": "2017-05-18T13:17:54.000Z",
  },
  Object {
    "depth": "11.3",
    "humanReadableLocation": "3.0 km A af Hvannadalshnjúk",
    "latitude": "64.017",
    "longitude": "-16.621",
    "quality": "99.0",
    "size": "0.1",
    "timestamp": "2017-05-18T12:01:24.000Z",
  },
  Object {
    "depth": "3.1",
    "humanReadableLocation": "11.4 km NNV af Álftavatni",
    "latitude": "63.959",
    "longitude": "-19.290",
    "quality": "99.0",
    "size": "0.9",
    "timestamp": "2017-05-18T11:47:00.000Z",
  },
  Object {
    "depth": "4.3",
    "humanReadableLocation": "3.7 km V af Þrengslum",
    "latitude": "64.003",
    "longitude": "-21.532",
    "quality": "99.0",
    "size": "0.2",
    "timestamp": "2017-05-18T11:30:31.000Z",
  },
  Object {
    "depth": "11.0",
    "humanReadableLocation": "3.7 km ASA  af Bárðarbungu",
    "latitude": "64.623",
    "longitude": "-17.461",
    "quality": "99.0",
    "size": "1.4",
    "timestamp": "2017-05-18T11:24:11.000Z",
  },
  Object {
    "depth": "2.7",
    "humanReadableLocation": "2.3 km ASA  af Goðabungu",
    "latitude": "63.635",
    "longitude": "-19.205",
    "quality": "99.0",
    "size": "0.8",
    "timestamp": "2017-05-18T10:26:04.000Z",
  },
  Object {
    "depth": "8.1",
    "humanReadableLocation": "18.4 km NV af Gjögurtá",
    "latitude": "66.271",
    "longitude": "-18.597",
    "quality": "99.0",
    "size": "0.8",
    "timestamp": "2017-05-18T10:00:16.000Z",
  },
  Object {
    "depth": "10.3",
    "humanReadableLocation": "18.6 km NV af Gjögurtá",
    "latitude": "66.270",
    "longitude": "-18.604",
    "quality": "99.0",
    "size": "0.6",
    "timestamp": "2017-05-18T09:32:44.000Z",
  },
  Object {
    "depth": "6.5",
    "humanReadableLocation": "18.3 km NV af Gjögurtá",
    "latitude": "66.269",
    "longitude": "-18.597",
    "quality": "99.0",
    "size": "1.7",
    "timestamp": "2017-05-18T09:30:13.000Z",
  },
  Object {
    "depth": "8.2",
    "humanReadableLocation": "18.5 km NV af Gjögurtá",
    "latitude": "66.273",
    "longitude": "-18.595",
    "quality": "99.0",
    "size": "2.2",
    "timestamp": "2017-05-18T09:29:08.000Z",
  },
  Object {
    "depth": "10.5",
    "humanReadableLocation": "18.9 km NV af Gjögurtá",
    "latitude": "66.275",
    "longitude": "-18.604",
    "quality": "99.0",
    "size": "0.8",
    "timestamp": "2017-05-18T08:58:12.000Z",
  },
  Object {
    "depth": "7.8",
    "humanReadableLocation": "18.2 km NV af Gjögurtá",
    "latitude": "66.269",
    "longitude": "-18.594",
    "quality": "99.0",
    "size": "1.1",
    "timestamp": "2017-05-18T08:57:40.000Z",
  },
  Object {
    "depth": "9.5",
    "humanReadableLocation": "18.2 km NV af Gjögurtá",
    "latitude": "66.269",
    "longitude": "-18.596",
    "quality": "99.0",
    "size": "1.5",
    "timestamp": "2017-05-18T08:48:36.000Z",
  },
  Object {
    "depth": "11.2",
    "humanReadableLocation": "18.7 km NV af Gjögurtá",
    "latitude": "66.272",
    "longitude": "-18.602",
    "quality": "99.0",
    "size": "2.5",
    "timestamp": "2017-05-18T08:46:04.000Z",
  },
  Object {
    "depth": "11.2",
    "humanReadableLocation": "18.1 km VNV af Gjögurtá",
    "latitude": "66.259",
    "longitude": "-18.608",
    "quality": "99.0",
    "size": "0.7",
    "timestamp": "2017-05-18T07:27:09.000Z",
  },
  Object {
    "depth": "0.1",
    "humanReadableLocation": "3.3 km ASA  af Álftavatni",
    "latitude": "63.848",
    "longitude": "-19.167",
    "quality": "99.0",
    "size": "0.1",
    "timestamp": "2017-05-18T07:15:20.000Z",
  },
  Object {
    "depth": "0.1",
    "humanReadableLocation": "14.6 km VNV af Gjögurtá",
    "latitude": "66.222",
    "longitude": "-18.566",
    "quality": "99.0",
    "size": "0.5",
    "timestamp": "2017-05-18T07:04:39.000Z",
  },
  Object {
    "depth": "7.1",
    "humanReadableLocation": "18.0 km NV af Gjögurtá",
    "latitude": "66.271",
    "longitude": "-18.587",
    "quality": "99.0",
    "size": "2.1",
    "timestamp": "2017-05-18T06:24:56.000Z",
  },
  Object {
    "depth": "7.8",
    "humanReadableLocation": "18.4 km NV af Gjögurtá",
    "latitude": "66.271",
    "longitude": "-18.597",
    "quality": "99.0",
    "size": "2.1",
    "timestamp": "2017-05-18T06:23:33.000Z",
  },
  Object {
    "depth": "10.4",
    "humanReadableLocation": "18.5 km NV af Gjögurtá",
    "latitude": "66.271",
    "longitude": "-18.601",
    "quality": "99.0",
    "size": "1.9",
    "timestamp": "2017-05-18T06:15:43.000Z",
  },
  Object {
    "depth": "7.6",
    "humanReadableLocation": "18.8 km NV af Gjögurtá",
    "latitude": "66.276",
    "longitude": "-18.599",
    "quality": "99.0",
    "size": "2.5",
    "timestamp": "2017-05-18T06:11:03.000Z",
  },
  Object {
    "depth": "8.3",
    "humanReadableLocation": "18.2 km NV af Gjögurtá",
    "latitude": "66.269",
    "longitude": "-18.594",
    "quality": "99.0",
    "size": "2.0",
    "timestamp": "2017-05-18T06:09:20.000Z",
  },
  Object {
    "depth": "6.7",
    "humanReadableLocation": "18.0 km NV af Gjögurtá",
    "latitude": "66.270",
    "longitude": "-18.589",
    "quality": "99.0",
    "size": "1.4",
    "timestamp": "2017-05-18T05:58:38.000Z",
  },
  Object {
    "depth": "8.2",
    "humanReadableLocation": "9.9 km SSA  af Árnesi",
    "latitude": "63.955",
    "longitude": "-20.202",
    "quality": "99.0",
    "size": "0.8",
    "timestamp": "2017-05-18T05:39:24.000Z",
  },
  Object {
    "depth": "9.6",
    "humanReadableLocation": "18.5 km NV af Gjögurtá",
    "latitude": "66.270",
    "longitude": "-18.601",
    "quality": "99.0",
    "size": "0.8",
    "timestamp": "2017-05-18T05:08:44.000Z",
  },
  Object {
    "depth": "8.5",
    "humanReadableLocation": "18.7 km NV af Gjögurtá",
    "latitude": "66.275",
    "longitude": "-18.598",
    "quality": "99.0",
    "size": "1.8",
    "timestamp": "2017-05-18T05:05:56.000Z",
  },
  Object {
    "depth": "8.0",
    "humanReadableLocation": "13.1 km A af Kistufelli",
    "latitude": "64.770",
    "longitude": "-16.933",
    "quality": "99.0",
    "size": "0.8",
    "timestamp": "2017-05-18T04:52:10.000Z",
  },
  Object {
    "depth": "1.3",
    "humanReadableLocation": "2.6 km SA  af Bárðarbungu",
    "latitude": "64.624",
    "longitude": "-17.490",
    "quality": "99.0",
    "size": "0.8",
    "timestamp": "2017-05-18T04:05:05.000Z",
  },
  Object {
    "depth": "6.4",
    "humanReadableLocation": "4.1 km NA af Bárðarbungu",
    "latitude": "64.662",
    "longitude": "-17.457",
    "quality": "99.0",
    "size": "1.2",
    "timestamp": "2017-05-18T03:42:09.000Z",
  },
  Object {
    "depth": "3.8",
    "humanReadableLocation": "14.4 km SA  af Kistufelli",
    "latitude": "64.673",
    "longitude": "-17.038",
    "quality": "99.0",
    "size": "0.5",
    "timestamp": "2017-05-18T03:38:14.000Z",
  },
  Object {
    "depth": "16.6",
    "humanReadableLocation": "19.2 km NV af Herðubreiðarfjöllum",
    "latitude": "65.462",
    "longitude": "-16.716",
    "quality": "99.0",
    "size": "0.4",
    "timestamp": "2017-05-18T03:29:14.000Z",
  },
  Object {
    "depth": "0.1",
    "humanReadableLocation": "4.9 km SSA  af Álftavatni",
    "latitude": "63.817",
    "longitude": "-19.203",
    "quality": "99.0",
    "size": "0.0",
    "timestamp": "2017-05-18T02:47:49.000Z",
  },
  Object {
    "depth": "6.5",
    "humanReadableLocation": "1.7 km ANA  af Bárðarbungu",
    "latitude": "64.645",
    "longitude": "-17.493",
    "quality": "99.0",
    "size": "1.2",
    "timestamp": "2017-05-18T02:03:07.000Z",
  },
  Object {
    "depth": "11.2",
    "humanReadableLocation": "5.8 km NA af Flatey",
    "latitude": "66.205",
    "longitude": "-17.753",
    "quality": "99.0",
    "size": "0.7",
    "timestamp": "2017-05-18T01:52:19.000Z",
  },
  Object {
    "depth": "10.9",
    "humanReadableLocation": "8.2 km V af Ólafsfirði",
    "latitude": "66.061",
    "longitude": "-18.833",
    "quality": "99.0",
    "size": "0.7",
    "timestamp": "2017-05-18T01:24:09.000Z",
  },
  Object {
    "depth": "0.1",
    "humanReadableLocation": "2.2 km V af Goðabungu",
    "latitude": "63.642",
    "longitude": "-19.295",
    "quality": "99.0",
    "size": "0.4",
    "timestamp": "2017-05-18T01:20:22.000Z",
  },
  Object {
    "depth": "0.1",
    "humanReadableLocation": "4.3 km VNV af Goðabungu",
    "latitude": "63.648",
    "longitude": "-19.335",
    "quality": "99.0",
    "size": "0.2",
    "timestamp": "2017-05-18T01:19:19.000Z",
  },
  Object {
    "depth": "22.0",
    "humanReadableLocation": "6.1 km SSA  af Básum",
    "latitude": "63.626",
    "longitude": "-19.435",
    "quality": "43.62",
    "size": "0.1",
    "timestamp": "2017-05-18T01:19:18.000Z",
  },
  Object {
    "depth": "0.1",
    "humanReadableLocation": "1.2 km S af Bárðarbungu",
    "latitude": "64.630",
    "longitude": "-17.523",
    "quality": "99.0",
    "size": "1.5",
    "timestamp": "2017-05-18T00:46:07.000Z",
  },
  Object {
    "depth": "0.1",
    "humanReadableLocation": "9.0 km V af Landmannalaugum",
    "latitude": "63.975",
    "longitude": "-19.241",
    "quality": "99.0",
    "size": "0.2",
    "timestamp": "2017-05-18T00:37:54.000Z",
  },
  Object {
    "depth": "8.1",
    "humanReadableLocation": "3.0 km N  af Bárðarbungu",
    "latitude": "64.668",
    "longitude": "-17.525",
    "quality": "99.0",
    "size": "2.2",
    "timestamp": "2017-05-17T23:22:42.000Z",
  },
  Object {
    "depth": "5.5",
    "humanReadableLocation": "1.6 km VSV af Hestfjalli",
    "latitude": "64.013",
    "longitude": "-20.703",
    "quality": "99.0",
    "size": "0.5",
    "timestamp": "2017-05-17T23:16:03.000Z",
  },
  Object {
    "depth": "3.6",
    "humanReadableLocation": "2.8 km NA af Hábungu",
    "latitude": "63.600",
    "longitude": "-19.066",
    "quality": "99.0",
    "size": "0.5",
    "timestamp": "2017-05-17T23:10:10.000Z",
  },
  Object {
    "depth": "10.9",
    "humanReadableLocation": "5.5 km ANA  af Flatey",
    "latitude": "66.190",
    "longitude": "-17.734",
    "quality": "38.39",
    "size": "0.8",
    "timestamp": "2017-05-17T22:25:00.000Z",
  },
  Object {
    "depth": "6.4",
    "humanReadableLocation": "7.1 km A af Grenivík",
    "latitude": "65.940",
    "longitude": "-18.025",
    "quality": "90.01",
    "size": "1.0",
    "timestamp": "2017-05-17T22:06:28.000Z",
  },
  Object {
    "depth": "5.0",
    "humanReadableLocation": "15.3 km A af Kistufelli",
    "latitude": "64.806",
    "longitude": "-16.891",
    "quality": "30.06",
    "size": "0.5",
    "timestamp": "2017-05-17T21:56:51.000Z",
  },
  Object {
    "depth": "4.0",
    "humanReadableLocation": "5.7 km VNV af Þjórsárbrú",
    "latitude": "63.956",
    "longitude": "-20.755",
    "quality": "90.03",
    "size": "1.1",
    "timestamp": "2017-05-17T18:40:58.000Z",
  },
  Object {
    "depth": "0.1",
    "humanReadableLocation": "6.2 km ASA  af Álftavatni",
    "latitude": "63.833",
    "longitude": "-19.117",
    "quality": "99.0",
    "size": "0.0",
    "timestamp": "2017-05-17T17:36:50.000Z",
  },
  Object {
    "depth": "17.6",
    "humanReadableLocation": "43.5 km N  af Siglufirði",
    "latitude": "66.535",
    "longitude": "-18.994",
    "quality": "99.0",
    "size": "1.7",
    "timestamp": "2017-05-17T17:07:44.000Z",
  },
  Object {
    "depth": "16.1",
    "humanReadableLocation": "17.5 km ASA  af Bárðarbungu",
    "latitude": "64.589",
    "longitude": "-17.180",
    "quality": "99.0",
    "size": "0.6",
    "timestamp": "2017-05-17T16:48:06.000Z",
  },
  Object {
    "depth": "11.8",
    "humanReadableLocation": "7.0 km NNA  af Flatey",
    "latitude": "66.221",
    "longitude": "-17.761",
    "quality": "99.0",
    "size": "0.6",
    "timestamp": "2017-05-17T16:43:55.000Z",
  },
  Object {
    "depth": "13.1",
    "humanReadableLocation": "6.2 km NNA  af Flatey",
    "latitude": "66.214",
    "longitude": "-17.767",
    "quality": "99.0",
    "size": "0.6",
    "timestamp": "2017-05-17T13:58:44.000Z",
  },
  Object {
    "depth": "9.6",
    "humanReadableLocation": "1.9 km VSV af Herðubreið",
    "latitude": "65.166",
    "longitude": "-16.383",
    "quality": "99.0",
    "size": "0.4",
    "timestamp": "2017-05-17T13:30:21.000Z",
  },
  Object {
    "depth": "4.2",
    "humanReadableLocation": "17.7 km ASA  af Bárðarbungu",
    "latitude": "64.605",
    "longitude": "-17.165",
    "quality": "99.0",
    "size": "0.4",
    "timestamp": "2017-05-17T13:30:00.000Z",
  },
  Object {
    "depth": "6.5",
    "humanReadableLocation": "2.7 km VNV af Kröfluvirkjun",
    "latitude": "65.723",
    "longitude": "-16.809",
    "quality": "99.0",
    "size": "0.9",
    "timestamp": "2017-05-17T13:14:27.000Z",
  },
  Object {
    "depth": "13.1",
    "humanReadableLocation": "6.8 km NA af Flatey",
    "latitude": "66.213",
    "longitude": "-17.743",
    "quality": "99.0",
    "size": "0.4",
    "timestamp": "2017-05-17T13:01:03.000Z",
  },
  Object {
    "depth": "10.1",
    "humanReadableLocation": "1.8 km VNV af Húsavík",
    "latitude": "66.046",
    "longitude": "-17.379",
    "quality": "99.0",
    "size": "0.4",
    "timestamp": "2017-05-17T12:42:56.000Z",
  },
  Object {
    "depth": "10.1",
    "humanReadableLocation": "5.8 km NA af Bárðarbungu",
    "latitude": "64.675",
    "longitude": "-17.435",
    "quality": "99.0",
    "size": "2.6",
    "timestamp": "2017-05-17T12:19:51.000Z",
  },
  Object {
    "depth": "4.1",
    "humanReadableLocation": "20.1 km SV af Húsafelli",
    "latitude": "64.549",
    "longitude": "-21.121",
    "quality": "99.0",
    "size": "1.3",
    "timestamp": "2017-05-17T11:46:56.000Z",
  },
  Object {
    "depth": "7.8",
    "humanReadableLocation": "1.9 km SV af Herðubreið",
    "latitude": "65.162",
    "longitude": "-16.377",
    "quality": "99.0",
    "size": "0.3",
    "timestamp": "2017-05-17T11:43:28.000Z",
  },
  Object {
    "depth": "0.1",
    "humanReadableLocation": "14.8 km A af Hábungu",
    "latitude": "63.575",
    "longitude": "-18.801",
    "quality": "99.0",
    "size": "0.0",
    "timestamp": "2017-05-17T10:27:01.000Z",
  },
  Object {
    "depth": "2.8",
    "humanReadableLocation": "5.8 km ANA  af Bárðarbungu",
    "latitude": "64.656",
    "longitude": "-17.412",
    "quality": "99.0",
    "size": "0.7",
    "timestamp": "2017-05-17T10:25:32.000Z",
  },
  Object {
    "depth": "5.8",
    "humanReadableLocation": "5.9 km VSV af Herðubreið",
    "latitude": "65.145",
    "longitude": "-16.454",
    "quality": "99.0",
    "size": "0.3",
    "timestamp": "2017-05-17T10:00:38.000Z",
  },
  Object {
    "depth": "5.7",
    "humanReadableLocation": "1.3 km N  af Dreka",
    "latitude": "65.050",
    "longitude": "-16.603",
    "quality": "99.0",
    "size": "0.4",
    "timestamp": "2017-05-17T09:46:15.000Z",
  },
  Object {
    "depth": "2.8",
    "humanReadableLocation": "6.7 km ANA  af Bárðarbungu",
    "latitude": "64.669",
    "longitude": "-17.403",
    "quality": "99.0",
    "size": "2.0",
    "timestamp": "2017-05-17T09:26:41.000Z",
  },
  Object {
    "depth": "0.6",
    "humanReadableLocation": "9.5 km NV af Hvannadalshnjúk",
    "latitude": "64.083",
    "longitude": "-16.803",
    "quality": "99.0",
    "size": "0.6",
    "timestamp": "2017-05-17T08:27:56.000Z",
  },
  Object {
    "depth": "0.9",
    "humanReadableLocation": "3.2 km SA  af Goðabungu",
    "latitude": "63.624",
    "longitude": "-19.197",
    "quality": "99.0",
    "size": "0.9",
    "timestamp": "2017-05-17T08:02:14.000Z",
  },
  Object {
    "depth": "0.1",
    "humanReadableLocation": "14.3 km A af Hábungu",
    "latitude": "63.602",
    "longitude": "-18.815",
    "quality": "99.0",
    "size": "0.0",
    "timestamp": "2017-05-17T07:43:39.000Z",
  },
  Object {
    "depth": "3.0",
    "humanReadableLocation": "5.8 km NA af Bárðarbungu",
    "latitude": "64.672",
    "longitude": "-17.431",
    "quality": "99.0",
    "size": "1.8",
    "timestamp": "2017-05-17T07:33:39.000Z",
  },
  Object {
    "depth": "5.1",
    "humanReadableLocation": "2.9 km SA  af Hrómundartindi",
    "latitude": "64.060",
    "longitude": "-21.158",
    "quality": "99.0",
    "size": "0.7",
    "timestamp": "2017-05-17T07:31:04.000Z",
  },
  Object {
    "depth": "4.0",
    "humanReadableLocation": "14.1 km S af Eiríksjökli",
    "latitude": "64.645",
    "longitude": "-20.368",
    "quality": "99.0",
    "size": "1.4",
    "timestamp": "2017-05-17T06:47:52.000Z",
  },
  Object {
    "depth": "6.7",
    "humanReadableLocation": "3.2 km NV af Upptyppingum",
    "latitude": "65.062",
    "longitude": "-16.295",
    "quality": "99.0",
    "size": "1.1",
    "timestamp": "2017-05-17T06:30:09.000Z",
  },
  Object {
    "depth": "0.1",
    "humanReadableLocation": "6.6 km A af Goðabungu",
    "latitude": "63.647",
    "longitude": "-19.117",
    "quality": "99.0",
    "size": "1.5",
    "timestamp": "2017-05-17T06:06:13.000Z",
  },
  Object {
    "depth": "3.2",
    "humanReadableLocation": "4.3 km SSV af Hrómundartindi",
    "latitude": "64.044",
    "longitude": "-21.241",
    "quality": "99.0",
    "size": "0.9",
    "timestamp": "2017-05-17T05:55:26.000Z",
  },
  Object {
    "depth": "7.2",
    "humanReadableLocation": "10.1 km ANA  af Þjórsárbrú",
    "latitude": "63.982",
    "longitude": "-20.479",
    "quality": "99.0",
    "size": "0.7",
    "timestamp": "2017-05-17T05:37:36.000Z",
  },
  Object {
    "depth": "6.1",
    "humanReadableLocation": "8.5 km NA af Flatey",
    "latitude": "66.226",
    "longitude": "-17.722",
    "quality": "99.0",
    "size": "0.2",
    "timestamp": "2017-05-17T05:26:53.000Z",
  },
  Object {
    "depth": "16.3",
    "humanReadableLocation": "9.8 km ASA  af Hamrinum",
    "latitude": "64.440",
    "longitude": "-17.636",
    "quality": "99.0",
    "size": "0.6",
    "timestamp": "2017-05-17T04:42:41.000Z",
  },
  Object {
    "depth": "4.6",
    "humanReadableLocation": "3.6 km SV af Hrómundartindi",
    "latitude": "64.052",
    "longitude": "-21.246",
    "quality": "99.0",
    "size": "2.1",
    "timestamp": "2017-05-17T04:38:47.000Z",
  },
  Object {
    "depth": "9.9",
    "humanReadableLocation": "6.5 km NA af Flatey",
    "latitude": "66.213",
    "longitude": "-17.754",
    "quality": "99.0",
    "size": "0.2",
    "timestamp": "2017-05-17T04:36:01.000Z",
  },
  Object {
    "depth": "10.8",
    "humanReadableLocation": "11.5 km NNA  af Grímsey",
    "latitude": "66.642",
    "longitude": "-17.942",
    "quality": "99.0",
    "size": "1.0",
    "timestamp": "2017-05-17T04:16:54.000Z",
  },
  Object {
    "depth": "10.9",
    "humanReadableLocation": "7.5 km NA af Flatey",
    "latitude": "66.220",
    "longitude": "-17.740",
    "quality": "99.0",
    "size": "0.0",
    "timestamp": "2017-05-17T04:15:31.000Z",
  },
  Object {
    "depth": "13.4",
    "humanReadableLocation": "6.6 km NNV af Bárðarbungu",
    "latitude": "64.689",
    "longitude": "-17.604",
    "quality": "99.0",
    "size": "0.4",
    "timestamp": "2017-05-17T03:57:37.000Z",
  },
  Object {
    "depth": "11.2",
    "humanReadableLocation": "6.8 km NNA  af Flatey",
    "latitude": "66.220",
    "longitude": "-17.767",
    "quality": "99.0",
    "size": "0.1",
    "timestamp": "2017-05-17T03:38:38.000Z",
  },
  Object {
    "depth": "11.4",
    "humanReadableLocation": "4.6 km SA  af Bárðarbungu",
    "latitude": "64.610",
    "longitude": "-17.460",
    "quality": "99.0",
    "size": "0.5",
    "timestamp": "2017-05-17T03:24:26.000Z",
  },
  Object {
    "depth": "3.9",
    "humanReadableLocation": "17.5 km S af Eiríksjökli",
    "latitude": "64.615",
    "longitude": "-20.351",
    "quality": "99.0",
    "size": "0.7",
    "timestamp": "2017-05-17T03:11:11.000Z",
  },
  Object {
    "depth": "11.3",
    "humanReadableLocation": "3.8 km SA  af Bárðarbungu",
    "latitude": "64.615",
    "longitude": "-17.475",
    "quality": "99.0",
    "size": "0.3",
    "timestamp": "2017-05-17T02:52:23.000Z",
  },
  Object {
    "depth": "8.0",
    "humanReadableLocation": "4.5 km NNV af Þjórsárbrú",
    "latitude": "63.967",
    "longitude": "-20.697",
    "quality": "99.0",
    "size": "0.2",
    "timestamp": "2017-05-17T02:14:40.000Z",
  },
  Object {
    "depth": "0.1",
    "humanReadableLocation": "3.8 km SA  af Bárðarbungu",
    "latitude": "64.619",
    "longitude": "-17.464",
    "quality": "99.0",
    "size": "0.3",
    "timestamp": "2017-05-17T00:53:41.000Z",
  },
  Object {
    "depth": "14.1",
    "humanReadableLocation": "32.8 km NNA  af Húsavík",
    "latitude": "66.330",
    "longitude": "-17.185",
    "quality": "99.0",
    "size": "0.2",
    "timestamp": "2017-05-17T00:50:31.000Z",
  },
  Object {
    "depth": "2.9",
    "humanReadableLocation": "4.9 km NNV af Grímsfjalli",
    "latitude": "64.448",
    "longitude": "-17.297",
    "quality": "99.0",
    "size": "0.6",
    "timestamp": "2017-05-17T00:21:33.000Z",
  },
  Object {
    "depth": "11.6",
    "humanReadableLocation": "6.9 km NA af Flatey",
    "latitude": "66.217",
    "longitude": "-17.751",
    "quality": "99.0",
    "size": "1.8",
    "timestamp": "2017-05-17T00:05:53.000Z",
  },
  Object {
    "depth": "12.5",
    "humanReadableLocation": "14.0 km N  af Grímsey",
    "latitude": "66.667",
    "longitude": "-17.982",
    "quality": "99.0",
    "size": "1.2",
    "timestamp": "2017-05-16T23:50:18.000Z",
  },
  Object {
    "depth": "16.5",
    "humanReadableLocation": "13.2 km N  af Húsavík",
    "latitude": "66.160",
    "longitude": "-17.383",
    "quality": "99.0",
    "size": "0.2",
    "timestamp": "2017-05-16T23:43:14.000Z",
  },
  Object {
    "depth": "13.1",
    "humanReadableLocation": "12.3 km N  af Grímsey",
    "latitude": "66.651",
    "longitude": "-17.978",
    "quality": "99.0",
    "size": "0.9",
    "timestamp": "2017-05-16T23:41:41.000Z",
  },
  Object {
    "depth": "8.6",
    "humanReadableLocation": "4.8 km N  af Bárðarbungu",
    "latitude": "64.684",
    "longitude": "-17.518",
    "quality": "99.0",
    "size": "0.0",
    "timestamp": "2017-05-16T23:37:03.000Z",
  },
  Object {
    "depth": "4.9",
    "humanReadableLocation": "14.7 km NNA  af Grímsey",
    "latitude": "66.654",
    "longitude": "-17.834",
    "quality": "99.0",
    "size": "0.8",
    "timestamp": "2017-05-16T23:23:05.000Z",
  },
  Object {
    "depth": "13.6",
    "humanReadableLocation": "13.8 km N  af Grímsey",
    "latitude": "66.666",
    "longitude": "-17.996",
    "quality": "99.0",
    "size": "0.7",
    "timestamp": "2017-05-16T23:09:02.000Z",
  },
  Object {
    "depth": "1.1",
    "humanReadableLocation": "8.3 km N  af Grímsey",
    "latitude": "66.615",
    "longitude": "-17.980",
    "quality": "99.0",
    "size": "0.9",
    "timestamp": "2017-05-16T23:00:14.000Z",
  },
  Object {
    "depth": "10.9",
    "humanReadableLocation": "7.0 km NA af Flatey",
    "latitude": "66.213",
    "longitude": "-17.736",
    "quality": "62.64",
    "size": "0.9",
    "timestamp": "2017-05-16T21:41:00.000Z",
  },
  Object {
    "depth": "4.8",
    "humanReadableLocation": "16.5 km SSV af Geirfugladrangi_á_Rneshr.",
    "latitude": "63.536",
    "longitude": "-23.383",
    "quality": "31.08",
    "size": "2.0",
    "timestamp": "2017-05-16T19:52:21.000Z",
  },
  Object {
    "depth": "5.6",
    "humanReadableLocation": "12.2 km ANA  af Kistufelli",
    "latitude": "64.816",
    "longitude": "-16.963",
    "quality": "30.21",
    "size": "0.1",
    "timestamp": "2017-05-16T19:24:39.000Z",
  },
  Object {
    "depth": "11.2",
    "humanReadableLocation": "7.1 km NA af Flatey",
    "latitude": "66.218",
    "longitude": "-17.747",
    "quality": "44.26",
    "size": "0.8",
    "timestamp": "2017-05-16T19:15:05.000Z",
  },
]
`;
